docs/features/metrics.md

```diff
@@ -2,28 +2,28 @@
 
 Traditional approaches of identifying students who are struggling with class is reactive; course staff wait for students to come to them to provide help, which can often be too late.
 
-The metrics feature seeks to be an proactive approach by actively identifying students who might be struggling in class through tracking of metrics that signify possible risks. Identifying at-risk students early would provide a better chance of getting them back on track in the course.
+The metrics feature seeks to be an proactive approach by actively identifying students who might be struggling in class through tracking of metrics that signify possible risks. Identifying students in need of attention early in the course would provide a better chance of getting them back on track in the course.
 
 
 ## Usage Flow
 
 We envision the feature to be used by the instructors in this order. They would:
 
 1. Set up their course and assignments as per usual
-1. Set up [Risk Metrics](#risk-metrics) at the start of the course
-1. Be notified of new at risk students in their [Watchlist](#watchlist) on their course page
+1. Set up [Student Metrics](#Student-Metrics) at the start of the course
```

Comment:
Seems like the link (#Student-Metrics) needs to be all lowercase (#student-metrics) for it to work.

Reply:
Thanks for pointing that out! Should be fixed now